src/ServerConfigParams.ts

```diff
@@ -307,6 +310,17 @@ export async function resolveServerConfig(
     ) {
         error(`RelayHub: no contract at address ${config.relayHubAddress}`);
     }
+
+    if (config.feesReceiver) {
+        const isFeesReceiverDeployed =
+            await contractInteractor.isContractDeployed(config.feesReceiver);
+        if (!isFeesReceiverDeployed) {
+            error(
+                `FeesReceiver: no contract at address ${config.feesReceiver}`
+            );
+        }
```

Comment:
If we perform this check, how can we set this configuration with the `workerAddress`? 🤔

Reply:
We would have to check if the fees receiver is different than the worker address, then perform the check.

Reply:
@antomor Actually, this check only makes sense if the feesReceiver is always a contract or the worker address. But what if the feesReceiver is set to an EOA? Is that also a valid use case? If it is, then this check must be removed.